modules/init-treesit.el

```diff
@@ -55,6 +38,24 @@
                 rust
                 scala
                 ))
-        (setq treesit-font-lock-level 4))))
+        (setq treesit-font-lock-level 4)))
+
+  (use-package tree-sitter-langs)
```

Comment:
These need to be guarded by treesit being available, too, or there are failures in 28 and 27. At least that's what I've seen locally.

Reply:
They are. This PR is about making `ci_tests` the top level guard. Only when it is not set the `if` from line 6 will execute. This is start of the `else` branch - i.e., what is designed for Emacs 28/27 (and Emacs 29 should it have not `treesit-available-p`).